content/docs/testing-farm.md

```diff
@@ -315,6 +315,27 @@ execute:
     how: tmt
 ```
 
+You can run rpminspect also using the CentOS Stream configuration. This should prepare you before opening CentOS Stream dist-git MRs.
+```yaml
+summary:
+    Fetch SRPM and check it with rpminspect
+discover:
+    how: shell
+    tests:
+      - name: rpminspect
+        test: rpminspect-centos -v -t VERIFY --profile=centos-stream-9-devel /tmp/*.src.rpm
```

Comment:
Any reason why to run rpminspect only on the SRPM? Binary packages are well supported as well (it's actually where rpminspect truly shines).

Reply:
good point! quite honestly, I wasn't aware of that

trying to figure out with TF/tmt if those are already available in the environment because there is no easy way to download them (by easy I mean I'd need to write an ugly shell script)